endpoints/get_websites-banners.md

```diff
@@ -115,14 +118,11 @@ GET `https://api.termly.io/v1/websites/banners?query=%5B%0A%20%20%7B%0A%20%20%20
     }
   ],
   "errors": [],
-  "paging": {
-    "count": 1,
-    "current_page": 1,
+  "scrolling": {
     "next_page": null,
     "previous_page": null,
-    "per_page": 25,
-    "total_count": 1,
-    "total_pages": 1
+    "group_size": 25,
+    "total_count": 1
```

Comment:
You really don’t want to update this one do you 🙂

Reply:
I've grown quite fond of it!